tests/testthat/test-response-check.R

```diff
@@ -72,8 +72,22 @@ test_that(desc = "OK when application type is json and status code is 200.",
 
 # Test whether package can pull some sample data
 # Source query data once to use later across tests
-prom <-
-  Prometheus$new(host = "http://demo.robustperception.io", port = 9090)
+
+# Create an object
+prom <- Prometheus$new(host = "http://demo.robustperception.io",
+                       port = 9090)
+
+test_that(desc = "Check that class functions source data.",
```

Comment:
The approach looks good, removes the need to call out to a service which may go down. I reckon we can roll out this pattern to the other main API calls.